lib/dk_buttons_and_leds/dk_buttons_and_leds.h

```diff
@@ -67,11 +67,19 @@ int dk_buttons_init(button_handler_t button_handler);
 
 /** @brief Read current button states
  *
- * @param button_state Bitmask of button states
- * @param has_changed Bitmask that shows wich buttons that have changed.
+ *  @param button_state Bitmask of button states
+ *  @param has_changed Bitmask that shows wich buttons that have changed.
  */
 void dk_read_buttons(u32_t *button_state, u32_t *has_changed);
 
+/** @brief Get current buttons state
+ *
+ *  Get buttons state from internal variable.
+ *
+ *  @return Bibmask of button states.
+ */
+u32_t dk_get_buttons(void);
```

Comment:
It seems to have very similar functionality to dk_read_buttons. It is really needed ?

Reply:
Read buttons makes also analyse about previous button state and it is costs a lot of cycles. If you wish to get only current state and do not want do change the "previous" state this is better way to go.

Reply:
Ok